airbyte-cdk/python/airbyte_cdk/sources/streams/core.py

```diff
@@ -27,6 +27,8 @@
 # AirbyteMessage: An AirbyteMessage. Could be of any type
 StreamData = Union[Mapping[str, Any], AirbyteMessage]
 
+JsonSchema = Dict[str, Any]
```

Comment:
nit: this could be a `Mapping[str, Any]` since the JsonSchema should be immutable